source/draw/Color.ooc

```diff
@@ -70,6 +71,7 @@ ColorUv: cover {
 		this v = color v
 	}
 	init: func@ (=u, =v)
+	// TODO: use valid default colors
```

Comment:
At some stage, someone have to convert to RGB before displaying on the monitor or CMYK for print.